Automattic/newspack-plugin
# feat: memberships block patterns

### All Submissions:






### Changes proposed in this Pull Request:

Adds block patterns to be used for content gating.

<img width="651" alt="image" src="https://user-images.githubusercontent.com/820752/233184409-4a2f1263-bceb-49a1-9e0b-f302441d156e.png">

| Pattern | Screenshot |
| --- | --- |
| Registration Wall | <img width="825" alt="image" src="https://user-images.githubusercontent.com/820752/233184281-faf96e38-f11e-4332-9dcd-2c1373b9c927.png"> |
| Donation Wall | <img width="827" alt="image" src="https://user-images.githubusercontent.com/820752/233183848-fe3a6d2b-7b3a-43f5-a679-7f7ab070e16d.png"> |
| Pay Wall with One Tier | <img width="840" alt="image" src="https://user-images.githubusercontent.com/820752/233184054-cba41527-331c-4d2f-af07-9c5201a1dbaa.png"> |

### How to test the changes in this Pull Request:

1. Check out this branch and draft a new page
2. Navigate to add a block pattern and confirm you see the "Memberships" category
3. Add all three blocks and confirm they render as expected

### Other information:
